openday_scavenger/api/puzzles/schemas.py

```diff
@@ -32,3 +34,7 @@ class PuzzleAccess(BaseModel):
 class ResponseTestCreate(BaseModel):
     number_visitors: int = 3000
     number_wrong_answers: int = 3
+
+
+class PuzzleJson(BaseModel):
+    puzzles: List[dict]
```

Comment:
I tried to add stricter validation here but it got really complicated because the uploaded data models are a mix of "create" and "update" models (and they can have IDs). It seemed much cleaner to decide whether it was an update or create based on the unstructured data and then push the validation later.